src/orca-jedi/increment/Increment.cc

```diff
@@ -25,11 +26,14 @@
 
 #include "ufo/GeoVaLs.h"
 
-#include "orca-jedi/errorcovariance/ErrorCovariance.h"
```

Comment:
Do we still need this error covariance class at all? I am not up with the state of the art on JEDI covariances. Are you using one from saber? I am guessing if the class was removed entirely I would need to make changes to the traits file.

Reply:
Yes this code is probably not needed as I've been encouraged to do the error covariance work with SABER. I was planning to delete it in number 4 of my pull request plan where I implement the SABER covariance. I can make sure the traits are still working then also.